requirements.txt

```
@@ -0,0 +1,3 @@
+numpy==1.16.2
```

And here :/ -> 
```
ERROR: opencv-python 4.1.2.30 has requirement numpy>=1.17.3, but you'll have numpy 1.16.2 which is incompatible.
```
So we should put 
```suggestion
numpy>=1.17.3
```